crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -703,8 +703,12 @@ fn process_room_properties(room_data: &v4::SlidingSyncRoom, room_info: &mut Room
     room_summary.joined_member_count = room_data.joined_count;
 
     if let Some(heroes) = &room_data.heroes {
-        room_summary.heroes =
-            heroes.iter().filter_map(|hero| hero.name.as_ref().cloned()).collect();
+        // It's not clear for Ruma, but the `heroes` field should be a collection of
+        // [`UserId`].
```

Comment:
Fixed in https://github.com/ruma/ruma/pull/1822

Reply:
Ruma's patch has been merged. We may need to update `Cargo.toml` and update this line.

Thanks for finding the issue!